tests/TestCase.php

```diff
@@ -28,4 +28,18 @@ public function createApplication()
 
         return $app;
     }
+
+    /**
+     * Disable valid_phone validator.
+     * @return $this
+     */
+    protected function withoutPhoneValidation()
+    {
+        Validator::extend('valid_phone', function ($attribute, $value, $parameters, $validator) {
+            // Skip validation because we can't validate a phone number on test creds
+            return true;
+        });
+
+        return $this;
```

Comment:
Is this un-extended with each new test run? If not, it'll be confusing to see that it runs per method but stays per each full testing run. In that case, it'd be better to import it as a trait sort of like the `withoutMiddleware` trait.  But if it is un-extended for each run, we're good. :+1: